syntax = "proto3";

package NGenerativeBoltalka.Proto;

message TPtuneEmbeddings {
    reserved 4;
    string Embeddings = 1;
    uint32 NumTokens = 2;
    uint32 Precision = 3;
    oneof Location {
        string YtPath = 5;
        string S3ObjectName = 6;
    }
}
